tests/resources/serializers/test_ui_serializer.py

```diff
@@ -31,7 +31,7 @@ def _add_affiliation_name(creatibutors):
 
 
 @pytest.fixture(scope="function")
-def full_to_dict_record(full_record_to_dict):
+def full_to_dict_record_point(full_record_to_dict):
```

Comment:
I'm not a fan of having two versions of this, one with points and one with polygons. I know there are some organizational issues around the test data, so others might have better ideas. But I think including both polygon and point in the full_dict_record, and then checking once that they serialize correctly would be simpler but still have the same effect in validating that polygon is working correctly.

Reply:
> I think including both polygon and point in the full_dict_record, and then checking once that they serialize correctly would be simpler but still have the same effect in validating that polygon is working correctly.

Great idea! I will work on that change.

Reply:
@tmorrell I've consolidated the test cases and fixtures.

Reply:
Looks great! I'll plan on merging in the next few days unless there is feedback from others.

Reply:
> Looks great! I'll plan on merging in the next few days unless there is feedback from others.

Hey Tom. Have you gotten any feedback from others about this fix?

Reply:
Nope, but I've merged it. Do you need this fix on a particular RDM version?

Reply:
> Nope, but I've merged it. Do you need this fix on a particular RDM version?

On V12 would be great.